doc/adr/0002-recipe-naming.md

```diff
@@ -39,7 +39,7 @@ public class DeleteKey extends Recipe {
 
 Recipes are currently organized hierarchically by package name. These package naming guidelines apply to both recipes defined in code as well as the name given to declarative recipes defined in YAML.
 
-* **DO** start every OpenRewrite recipe package with with `org.openrewrite.<LANGUAGE>`.
+* **DO** start every OpenRewrite recipe package with `org.openrewrite.<LANGUAGE>`.
```

Comment:
Sometimes words are intentionally doubled, this doesn't appear to be such an instance